docs/conf/prague/2022/news/get-ready.rst

```diff
@@ -0,0 +1,71 @@
+:template: {{year}}/generic.html
+
+.. post:: Sept 1, 2022
+   :tags: {{year}}, {{shortcode}}-{{year}}, tickets
+
+Get Ready for the conference!
+=============================
+
+Hey everyone!
+
+We're just over a week away from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 3 days of content, collaboration, and conversation.
+
+We have a few reminders before we welcome you all next Sunday:
+
+* Ticket sales close next week, so grab yours now!
+* We have a great Welcome Wagon, which has a guide, as well as introductory tours of our online platform
+* There are lots of ways to get involved at the conference on the day-of, which we list out below so you can prep a bit for them.
+
+Ticket sales close on **September 8th**
+---------------------------------------
+
+To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **September 8th at 11:59PM Central European Time**.
+If you haven't purchased your ticket yet we invite you to `get your tickets soon <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.
+
+For those of you who already purchased your tickets, you will get an email with our online conference platform (Hopin) registration link on September 5th. We will also send a reminder on September 7th, in case you missed the first email.
+This will give you the opportunity to set up your Hopin account and troubleshoot any access issues that you might have.
+
+If you have a ticket and don't get an email from us by September 7th, please email us and we'll help you out!
+
+Welcome Wagon guide and platform tour
+-------------------------------------
+
+Each year we create a guide to help our attendees get the most out of the conference.
+Following the success of last year's conferences and based on your feedback, we expanded `our Welcome Wagon guide <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/welcome-wagon/>`_ to help you get the most out of our event this year.
+
+Another valuable thing that we're doing this year is the Welcome Wagon team will hold **live orientation sessions and platform tours** for our virtual platform, Hopin. **There will be 2 sessions on Sunday, and 1 on Monday morning.**  This will give you a chance to walk through the virtual event and ask all your questions ahead of the conference.
```

Comment:
```suggestion
Another valuable thing that we're doing this year is the Welcome Wagon team will hold **live orientation sessions and platform tours** for our virtual platform, Hopin. **There will be two sessions on Sunday, and one on Monday morning.**  This will give you a chance to walk through the virtual event and ask all your questions ahead of the conference.
```